packages/react-wildcat-handoff/src/utils/defaultTemplate.js

```diff
@@ -41,7 +42,19 @@ module.exports = function defaultTemplate(cfg) {
     </head>
     <body>
         <div id="${reactRootElementID}">${html}</div>
-
+        ${__PROD__ && serviceWorker ? `
```

Comment:
If `__PROD__` is false we want to run the `// Remove any registered service workers (dev mode only)` code, even if serviceWorker is `true` don't we?